conftest.py

```diff
@@ -1,35 +1,38 @@
-import pytest
-import time
-from selenium import webdriver
-from utils.aws_client import aws_client
-from utils.desired_capabilities import DesiredCapabilities
-import logging
-
-def pytest_addoption(parser):
-    parser.addoption(
-        '--browser', action='store', default='chrome', help='Browser to run tests'
-    )
-

```

Comment:
@appsmadeforu @arati-patil23 reformatted the file using dos2unix and fixed logging module issue